### PR TITLE
XIVY-11980 integrate variables editor

### PR DESCRIPTION
**Pull request by ivy-lgi**

### CODE REVIEW

File: playwright/tests/page-objects/variables-editor.ts [hunk 1]
```diff
@@ -12,43 +12,30 @@ export class VariablesEditor extends Editor {
...
 
   async hasKey(key: string) {
-    const field = this.viewFrameLoactor().locator(`vscode-text-field[grid-column="1"][current-value="${key}"]`);
+    const field = this.viewFrameLoactor().locator(`//td/div/span[contains(text(), '${key}')]`);
     await expect(field).toBeVisible();
   }
 
   async hasValue(value: string) {
-    const field = this.viewFrameLoactor().locator(`vscode-text-field[grid-column="2"][current-value="${value}"]`);
+    const field = this.viewFrameLoactor().locator(`//td/div[contains(text(), '${value}')]`);
```
Comment (ivy-lli) [inline playwright/tests/page-objects/variables-editor.ts:27]: not such a big fan of xpath locators...
I think it should also be possible to write this with CSS selectors

**Review by ivy-lli — APPROVED** [review]
Looks good. 
except the xpath stuff 🙈

File: playwright/tests/page-objects/page-object.ts [hunk 2]
```diff
@@ -3,14 +3,18 @@ import { Locator, Page, expect } from '@playwright/test';
 export class PageObject {
   constructor(readonly page: Page) {}
 
-  async executeCommand(command: string) {
+  async executeCommand(command: string, ...options: Array<string>) {
```
Comment (ivy-lmu) [inline playwright/tests/page-objects/page-object.ts:6]: is there a use case for more than one option?

**Review by ivy-lmu — APPROVED** [review]

File: playwright/tests/page-objects/page-object.ts (hunk 2 above)
Comment (ivy-lgi) [reply playwright/tests/page-objects/page-object.ts:6]: Yes, with an improvement I will make in the next commit, where I simply call `this.provideUserInput(option);` in the loop over `options`, something like:
https://github.com/axonivy/vscode-extensions/blob/38ecdbfc585e4396ae31e73f2a26df735b02d18b/playwright/tests/page-objects/explorer-view.ts#L76-L80

can be replaced with:
`await this.executeCommand('Axon Ivy: New Project', projectName, '', '', '');`

Comment (ivy-lmu) [reply playwright/tests/page-objects/page-object.ts:6]: very good 👍

**Review by ivy-lli — APPROVED** [review]

**Pull request merged**